ci/plugins/mzcompose/hooks/command

```diff
@@ -51,6 +51,14 @@ if ! mzcompose --mz-quiet list-workflows | grep -q "$service"; then
     mzcompose up -d --scale "$service=0" "$service"
 fi
 
+ci_uncollapsed_heading ":docker: Fetching binaries for coverage"
+mzcompose create
+mkdir -p coverage/
+chmod 777 coverage/
+mzcompose cp materialized:/usr/local/bin/environmentd coverage/ || true
```

Comment:
Can we expect those commands to always succeed? Why the `|| true`  ?

Reply:
I added a comment, not all tests contain all of the containers, thus we can't expect them to succeed